common/config/api.go

```diff
@@ -14,6 +15,11 @@ func APIFlags() []cli.Flag {
 			Value:  9090,
 			EnvVar: ENVVAR_PORT,
 		},
+		cli.DurationFlag{
+			Name:  FLAG_JOB_EXPIRY,
+			Value: ENVVAR_JOB_EXPIRY,
```

Comment:
This is the default, not the envvar

Reply:
So it should be like time.Hour * 24 or what should value you be. That's what I had it before.

Reply:
no, it should be `Value: DEFAULT_JOB_EXPIRY`, and there should exist 
```
const (
    DEFAULT_JOB_EXPIRY = time.Hour * 24
```